functions/packages.bash

```diff
@@ -546,28 +555,28 @@ nginx_setup() {
 
   echo "$(timestamp) [openHABian] Configuring Nginx network options... "
   cond_echo "Obtaining public IP address... "
-  if pubIP=$(dig -4 +short myip.opendns.com @resolver1.opendns.com | tail -1); then echo "$pubIP"; fi
-  if [ -z "$pubIP" ]; then
-    if pubIP=$(dig -4 TXT +short o-o.myaddr.l.google.com @ns1.google.com|tr -d '"'); then echo "$pubIP"; else echo "FAILED"; return 1; fi
+  if pubIP="$(dig -4 +short myip.opendns.com @resolver1.opendns.com | tail -1)"; then echo "$pubIP"; fi
+  if [[ -z $pubIP ]]; then
```

Comment:
helper function

Reply:
OK, I get it. I will create a helper function.

Reply:
done